ss/sqlite/db.go

```diff
@@ -53,11 +53,25 @@ type Database struct {
 }
 
 func New(dataDir string) (*Database, error) {
-	db, err := sql.Open(driverName, filepath.Join(dataDir, dbName))
+	db, err := sql.Open(driverName, filepath.Join(dataDir, dbName)+"?cache=shared")
 	if err != nil {
 		return nil, fmt.Errorf("failed to open sqlite DB: %w", err)
 	}
 
+	// TODO: Make fully configurable
+	pragmas := []string{
+		`PRAGMA journal_mode=WAL;`,
+		`PRAGMA synchronous=NORMAL;`,
+		`PRAGMA cache_size=-32000;`,
+		`PRAGMA auto_vacuum=FULL;`,
+		`PRAGMA temp_store=MEMORY;`,
+		`PRAGMA mmap_size=30000000000;`,
```

Comment:
Set to 30 GB, may reduce

Reply:
What's the main benefit of using mmap for sqlite as a state store?

Reply:
Updated